modules/cnf-intel-n3000-optimize-data-plane.adoc

```diff
@@ -0,0 +1,32 @@
+// CNF-1498 Validate and Document Intel SRO and SRIOV FEC Operator
+// Module included in the following assemblies:
+//
+// *cnf-optimize-data-performance-n3000.adoc
+
+[id="cnf-intel-n3000-optimize-data-plane_{context}"]
+= Understanding Intel hardware accelerator cards for OpenShift Container Platform
+
+Hardware accelerator cards from Intel accelerate 4G/LTE and 5G Virtualized Radio Access Networks (vRAN) workloads. This in turn increases the overall compute capacity of a commercial, off-the-shelf platform.
+
+.Intel FPGA PAC N3000
```

Comment:
Please change this block title and the following block title to a regular title, such as "== Intel FPGA..." If you do not want the heading to appear in the TOC, you can set it as a [discrete heading](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#discrete-headings) as shown in the team doc guidelines.